### ISSUE TITLE
Multiple python-based applications broke on nixos-unstable within the past two weeks.

### ISSUE THREAD

**Issue by jansol** [NixOS/nixpkgs#168504]
### Describe the bug
At least tauon and whipper stopped working sometime in the past two weeks due to not finding some dependencies at runtime. The packages themselves did not get updated in that time.

### Steps To Reproduce
1. install whipper
2. try running it

### Expected behavior
Whipper prints its help text

### Actual behavior
```
Traceback (most recent call last):
  File "/nix/store/msx7lpfscdnvkh3zccwz4wcdhn0g36dh-whipper-0.10.0/bin/.whipper-wrapped", line 6, in <module>
    from whipper.command.main import main
  File "/nix/store/msx7lpfscdnvkh3zccwz4wcdhn0g36dh-whipper-0.10.0/lib/python3.9/site-packages/whipper/__init__.py", line 5, in <module>
    from pkg_resources import (get_distribution,
ModuleNotFoundError: No module named 'pkg_resources'
```

### Notify maintainers
@emilazy
@FRidh

### Metadata
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.17.2, NixOS, 22.05 (Quokka), 22.05pre368735.33772708c6d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by mannp** [comment]
Tauon broke for me too

```
Install directory: /nix/store/pis9mnrqlx177pkbfws8msv45r7i0372-tauon-7.1.3/share/tauon
Warning: Python module natsort not found
Traceback (most recent call last):
  File "/nix/store/pis9mnrqlx177pkbfws8msv45r7i0372-tauon-7.1.3/bin/.tauon-wrapped", line 247, in <module>
    exec(main)
  File "/nix/store/pis9mnrqlx177pkbfws8msv45r7i0372-tauon-7.1.3/share/tauon/t_modules/t_main.py", line 746, in <module>
    import mutagen
ModuleNotFoundError: No module named 'mutagen'
```

```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.16.19, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - channels(root): `"home-manager, nixos, nixos-unstable-small"`
 - channels(local): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by SuperSandro2000** [comment]
That is probably some outfall from me/us cleaning up dependencies of core packages which where there no longer used and people relied on them being present.

**Comment by SuperSandro2000** [comment]
Both whipper and tauon works on master. It should be available in nixos-unstable in a few days.

**Comment by mweinelt** [comment]
Please reread the error description more carefully. It says the programs fail at runtime, not build time. I fixed whipper in #168634 and pinged you for tauon, not without reason.

```
❯ ./result/bin/tauon
Dev mode, ignoring single instancing
No previous window state
Tauon Music Box v7.1.3
Copyright 2015-2022 Taiko2k captain.gxj@gmail.com

Window size: [1120, 600]
GTK theme is: adwaita
Running from installed location
User files location: /home/hexa/.local/share/TauonMusicBox
Clearing old cache directory
/home/hexa/.cache/TauonMusicBox
Install directory: /nix/store/pis9mnrqlx177pkbfws8msv45r7i0372-tauon-7.1.3/share/tauon
Warning: Python module natsort not found
Traceback (most recent call last):
  File "/nix/store/pis9mnrqlx177pkbfws8msv45r7i0372-tauon-7.1.3/bin/.tauon-wrapped", line 247, in <module>
    exec(main)
  File "/nix/store/pis9mnrqlx177pkbfws8msv45r7i0372-tauon-7.1.3/share/tauon/t_modules/t_main.py", line 746, in <module>
    import mutagen
ModuleNotFoundError: No module named 'mutagen'

**Comment by SuperSandro2000** [comment]
The default output of mutagen is wrong. Fixing.
